main/test/flix/Test.Exp.ExtMatch.flix

```diff
@@ -1,18 +1,3 @@
-/*
- * Copyright 2025 Jakob Schneider Villumsen
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
```

Comment:
Why the deletion? Why not just add yourself to the list? :)

Reply:
We don't have copyright headers in test files.

Reply:
Why? Tja...